stages/org.osbuild.ostree.deploy.container

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/python3
+"""
+Deploy an OStree commit
+
+Create an OSTree deployment[1] for a given container image input
+"""
+
+import os
+import sys
+
+import osbuild.api
+from osbuild.util import containers, ostree
+from osbuild.util.mnt import MountGuard
+
+CAPABILITIES = ["CAP_MAC_ADMIN"]
+
+SCHEMA_2 = """
+"options": {
+  "additionalProperties": false,
+  "required": ["osname", "target_imgref"],
+  "properties": {
+    "mounts": {
+      "description": "Mount points of the final file system",
+      "type": "array",
+      "items": {
+        "description": "Description of one mount point",
+        "type": "string"
+      }
+    },
+    "osname": {
+      "description": "Name of the stateroot to be used in the deployment",
+      "type": "string"
+    },
+    "kernel_opts": {
+      "description": "Additional kernel command line options",
+      "type": "array",
+      "items": {
+        "description": "A single kernel command line option",
+        "type": "string"
+      }
+    },
+    "target_imgref": {
+      "description": "imageref used as the source of truth for updates",
+      "type": "string"
+    },
+    "rootfs": {
+      "description": "Identifier to locate the root file system",
+      "type": "object",
+      "oneOf": [{
+        "required": ["uuid"]
+      }, {
+        "required": ["label"]
+      }],
+      "properties": {
+        "label": {
+          "description": "Identify the root file system by label",
+          "type": "string"
+        },
+        "uuid": {
+          "description": "Identify the root file system by UUID",
+          "type": "string"
+        }
+      }
+    }
+  }
+},
+"inputs": {
+  "type": "object",
+  "additionalProperties": false,
+  "required": ["images"],
+  "properties": {
+    "images": {
+      "type": "object",
+      "description": "Container Image to deploy",
+      "additionalProperties": true
+    },
+    "manifest-lists": {
+      "type": "object",
+      "description": "Optional manifest lists to merge into images. The metadata must specify an image ID to merge to.",
+      "additionalProperties": true
+    }
```

Comment:
This doesn't apply here does it?  What this will do if specified is it will add a manifest list to the container `dir` when preparing it for the input, but I don't think it will have any effect on the final image.  Better remove it from the input schema.

Reply:
I think we were mostly copying the [skopeo stage input schema](https://github.com/osbuild/osbuild/blob/dd2aa57987edf9309c3fc70e5e673473eab13674/stages/org.osbuild.skopeo#L27-L31) here. The [underlying code](https://github.com/osbuild/osbuild/blob/dd2aa57987edf9309c3fc70e5e673473eab13674/osbuild/util/containers.py#L46-L52) does have some processing of manifest list stuff, but it looks like it's conditional so maybe not strictly required?

If you say it's not needed then maybe we can get rid of it?

Reply:
Removed.